components/CatCard.tsx

```diff
@@ -165,19 +169,17 @@ export const CatCard = ({
     <>
       {!shouldEdit ? (
         <Card style={styles.mainContainer}>
-          <Portal>
-            <Modal visible={visible} onDismiss={hideModal}>
-              <NucaModal
-                leftButtonHandler={hideModal}
-                rightButtonHandler={deleteCatCallback}
-                leftButtonMessage={'Renunță'}
-                rightButtonMessage={'Șterge'}
-                leftButtonIcon={'cancel'}
-                rightButtonIcon={'close'}
-                caption={'Ești sigur că vrei să ștergi?'}
-              />
-            </Modal>
-          </Portal>
+          <NucaModal
+            leftButtonHandler={hideModal}
+            rightButtonHandler={deleteCatCallback}
+            leftButtonMessage={'Renunță'}
+            rightButtonMessage={'Șterge'}
+            leftButtonIcon={'cancel'}
+            rightButtonIcon={'close'}
+            caption={'Ești sigur că vrei să ștergi?'}
+            visible={visible}
+            onDismiss={hideModal}
+          />
```

Comment:
not urgent, but maybe we should refactor this component so that we won't have so many `leftButton...`/`rightButton...` props.

Maybe something like this:
```jsx
<NucaModal
    leftButton={ /* add left button props here */ }
    rightButton={ /* add right button props here */ } 
    {...theRestOfProps} />
```